clusterloader2/testing/load/config.yaml

```diff
@@ -89,12 +119,12 @@ steps:
     Method: APIResponsivenessPrometheus
     Params:
       action: start
-  - Identifier: PodStartupLatency
+  - Identifier: CreatePhasePodStartupLatency
```

Comment:
Please remember to update https://github.com/kubernetes/perf-tests/blob/master/perfdash/config.go